### ISSUE TITLE
The compiled source code cannot run properly.   ort.min.js:1802 wasm streaming compile failed: LinkError: WebAssembly.instantiate(): Import #36 module="a" function="K": function import requires a callable

### ISSUE THREAD

**Issue by sunny-fun** [kevmo314/magic-copy#100]
Hello, I downloaded the source code of your open-source project magic-copy from releases/tag/v1.2.0 and installed dependencies locally. After running `npm run watch` and adding the plugin to Google Chrome, I found that it doesn't work properly. It shows an error like the one in the screenshot below.
![Snipaste_2024-02-19_14-00-40](https://github.com/kevmo314/magic-copy/assets/55755493/b155e449-94ec-4fc9-934e-97be5bcbe249)

However, if I download the compiled `magic-copy-chrome.zip` from releases/tag/v1.2.0 and install it directly in the browser, it works fine. I also checked the format of `onnxruntime-web` inside `sandbox.js` and found that they are different. Could this be an issue with the version of `onnxruntime-web`? I tried upgrading to the latest version of `onnxruntime-web`, but the issue persists.
![Snipaste_2024-02-19_14-06-28](https://github.com/kevmo314/magic-copy/assets/55755493/32187760-e220-4845-aa62-3c5d1f048335)

**Comment by kevmo314** [comment]
Yeah this sounds like something broken with the onnxruntime-web API. Does it work at 08d78db9a4aafb89302b0a6d5e813a50672fc2a6? Just one before the onnxruntime-web version bump.
